gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/OrganizationServiceImpl.java

```
@@ -87,6 +87,24 @@ public OrganizationEntity findById(String organizationId) {
         }
     }
 
+    @Override
+    public OrganizationEntity update(ExecutionContext executionContext, final UpdateOrganizationEntity organizationEntity) {
+        OrganizationEntity updatedOrganization = null;
+        try {
+            if (null != organizationEntity) {
```

We could throw an error before try-catch block if organizationEntity is null ?